src/main/java/com/checkmarx/flow/dto/report/GetResultsReport.java

```diff
@@ -10,63 +10,63 @@
 import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
 
-import java.util.HashMap;
+import java.util.EnumMap;
 import java.util.Map;
 
```

Comment:
I would have change the class name and reduce the initial 'Get' since a good convention for java classes are composed of nouns

Reply:
Changed to ScanResultsReport.